Autopilot/Path Manager/PathManager.c

```diff
@@ -340,7 +340,23 @@ char followWaypoints(PathData* currentWaypoint, float* position, float heading,
 
             *sp_Heading = (int)followStraightPath((float*)&waypointDirection, (float*)targetCoordinates, (float*)position, heading);
         }
-        else{
+        else{            
```

Comment:
I'm with you @iehink , not following your logic @chrishajduk84. It should still work normally if there are no hold waypoints.